src/Health/HealthController.ts

```diff
@@ -86,7 +86,7 @@ export class HealthController {
     return blockchainInfo
   }
 
-  async getWalletInfo(): Promise<WalletInfo> {
+  private async getWalletInfo(): Promise<WalletInfo> {
```

Comment:
I was trying to review this PR from my phone earlier, but this change confused me because it's not related to the changes here so I had to wait until I could get to my laptop.

You could fix things like this in a separate PR next time, just a tip to make things simpler for reviewers :)